Provider.react.js

```diff
@@ -1,6 +1,47 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import LocaleContext from './Context';
+import {getHostName} from 'virtual-tour-viewer/src/utils/UIHelper';
+
+async function isLocalHost() {
+    let hostname = !window.origin ? window.location.origin : window.origin;
+    return hostname.includes('localhost');
+}
+
+async function getLabelsFromHavrApi(locale) {
+    let taxonomyName = 'spaces';
+    // defaulting to en-US in case locale isn't present
+    locale = locale ? locale : 'en-US';
+    let tardisResponse;
+    if (await isLocalHost()) {
+        tardisResponse = await fetch(
+            `http://localhost:8080/havr-api/v2/taxonomies/${taxonomyName}/${locale}`
+        );
+        // if the viewer is running locally without havr-api, hit svc-stage instead
+        if (!tardisResponse.status !== '200') {
+            tardisResponse = await fetch(
+                `https://svc-stage.homeaway.com/havr-api/v2/taxonomies/${taxonomyName}/${locale}`
```

Comment:
Do not hardcode, use getHostName() to get host and append rest of the endpoint path.